titus-server-runtime/src/main/java/com/netflix/titus/runtime/jobmanager/JobManagerCursors.java

```diff
@@ -46,7 +49,10 @@ private JobManagerCursors() {
 
     /**
      * Compare two job entities by the creation time (first), and a job id (second).
+     *
+     * @deprecated Use core model entities.
```

Comment:
minor: I think this may deserve more consideration. Moving gateway and federation to the core models will introduce a bunch of parsing between types that is not adding much value to these layers since they are mostly only dealing with grpc models.

Did you consider doing all the pagination for master at the grpc layers only, and doing all the core -> grpc model conversions before applying pagination?

Reply:
To be clear, I think this is OK for now as is, but before committing one way or another for the future, we should discuss this more.

Reply:
I think that this question is usually answered by how much business logic exists in the higher tiers. If the responsibility of a higher tier is to have business logic then you may want to convert into a data model that makes it easier to perform that business logic. If the purpose is just to proxy passthrough very quickly then a conversion makes less sense and the data should stay in its transport specific form.